src/sonic-config-engine/portconfig.py

```diff
@@ -37,7 +37,7 @@
 BRKOUT_MODE = "default_brkout_mode"
 CUR_BRKOUT_MODE = "brkout_mode"
 INTF_KEY = "interfaces"
-OPTIONAL_HWSKU_ATTRIBUTES = ["fec", "autoneg", "subport"]
```

Comment:
@vivekrnv What about platforms that don't have hwsku.json and have only platform.json? How will they honor the role?